src/panels/lovelace/common/process-config-entities.ts

```diff
@@ -1,8 +1,8 @@
 // Parse array of entity objects from config
 import { isValidEntityId } from "../../../common/entity/valid_entity_id";
-import { EntityConfig } from "../entity-rows/types";
+import { EntityConfig, LovelaceRowConfig } from "../entity-rows/types";
 
-export const processConfigEntities = <T extends EntityConfig>(
+export const processConfigEntities = <T extends LovelaceRowConfig>(
```

Comment:
This is not correct? We use this function for Glance etc? that is not extending `LovelaceRowConfig`? Should be `EntityConfig`?

Reply:
It should be both then